.gitignore

```diff
@@ -1,5 +1,5 @@
 _site
-*.pyc
+*.py[cod]
```

Comment:
I googled "sad cod" and it turns out it's mostly a popular shooter game. Who knew.

Reply:
Just because I was ignorant of what `*.py[cod]` meant or did.  You learn something new everyday!

https://stackoverflow.com/questions/18729510/what-is-the-difference-between-pycod-and-pyc-in-gitignore-notation